aci/data_source_aci_bfdmhifp.go

```diff
@@ -0,0 +1,69 @@
+package aci
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/ciscoecosystem/aci-go-client/v2/client"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func dataSourceAciBfdMultihopInterfaceProfile() *schema.Resource {
+	return &schema.Resource{
+		ReadContext:   dataSourceAciBfdMultihopInterfaceProfileRead,
+		SchemaVersion: 1,
+		Schema: AppendBaseAttrSchema(AppendNameAliasAttrSchema(map[string]*schema.Schema{
+			"logical_interface_profile_dn": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"annotation": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
```

Comment:
I think optional in data sources is not needed and can be removed for all attributes.

Reply:
removed.